src/spikeinterface/core/sortinganalyzer.py

```diff
@@ -943,20 +943,41 @@ def get_saved_extension_names(self):
     def get_extension(self, extension_name: str):
         """
         Get a AnalyzerExtension.
+
         If not loaded then load is automatic.
 
         Return None if the extension is not computed yet (this avoids the use of has_extension() and then get it)
 
+        Parameters
+        ----------
+        extension_name: str
+            The extension name.
+            This also accept the or logic with the "|".
+            For instance get_extension("templates|fast_templates") will try to get "templates" extension, if not
+            exists then try to get "fast_templates" extension and if not exists return None.
+
+        Returns
+        -------
+        ext_instance: AnalyzerExtension | None
+            Return the extension instance if possible ortherwise None.
```

Comment:
```suggestion
        Parameters
        ----------
        extension_name: str
            The extension name.
            This also accept the or logic with the "|".
            For instance get_extension("templates|fast_templates") will try to get the "templates" extension, if it does not
            exist then it will try to get the "fast_templates" extension and if it does not exist  it will return None.

        Returns
        -------
        ext_instance: AnalyzerExtension | None
            Return the extension instance if possible otherwise None.
```